go/test/endtoend/tabletmanager/tablet_health_test.go

```diff
@@ -195,6 +195,136 @@ func TestHealthCheck(t *testing.T) {
 	killTablets(t, rTablet)
 }
 
+// TestHealthCheckSchemaChangeSignal tests the tables and views, which report their schemas have changed in the output of a StreamHealth.
+func TestHealthCheckSchemaChangeSignal(t *testing.T) {
```

Comment:
can this test be added to go/vt/vttablet/endtoend will that be any different? just trying to keep all the related test in one package.

Reply:
I thought of that too, but I can't since there we are using vttest. We can't enable and disable views easily without creating a new cluster entirely.